src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -484,4 +485,22 @@ BOOST_AUTO_TEST_CASE(PReLUFunctionTest)
   CheckPReLUGradientCorrect(activationData, desiredGradient);
 }
 
+/**
+ * Basic test of the swish function.
+ */
+BOOST_AUTO_TEST_CASE(SwishFunctionTest)
+{
+  const arma::colvec desiredActivations("-0.238405 3.07466 4.45055 \
```

Comment:
Can you comment on where this values come from?

Reply:
I calculated these swish function values using the python interpreter using the column vector `activationData` values as input data.

Reply:
Okay, sounds good, would be great if you could add a comment something like this: https://github.com/mlpack/mlpack/blob/master/src/mlpack/tests/logistic_regression_test.cpp#L41

Reply:
Added the comment with a new fixup commit.